webserver/http2/src/main/java/io/helidon/webserver/http2/Http2ServerResponse.java

```diff
@@ -161,10 +182,19 @@ public long bytesWritten() {
     public ServerResponseHeaders headers() {
         return headers;
     }
+    @Override
+    public ServerResponseHeaders trailers() {
```

Comment:
I think we should return just `Headers` instead of `ServerResponseHeaders`, as the methods on it are not relevant for trailers

Reply:
Changed to `ServerResponseTrailers` direct extension of writeable headers